playbooks/roles/monit/templates/monitrc.j2

```diff
@@ -5,9 +5,11 @@ set idfile /var/lib/monit/id
 set statefile /var/lib/monit/state
 
 set httpd port 2812 and
-    use address localhost  # only accept connection from localhost
-    allow localhost        # allow localhost to connect to the server
+    use address localhost  # Bind monit to localhost
 
+    # Allow any ip to connect to monit. Binding monit to localhost
+    # and requiring basic auth (through nginx) should be sufficient to protect this interface.
+    allow 0.0.0.0/0.0.0.0 # allow any ip to connect to monit.
```

Comment:
Thanks! That makes sense. I agree with your rationale here. I also tested that the monit bind is sufficiently scoped that if ufw/iptables weren't there you still can't connect from an external host.